moveit_core/collision_detection/src/collision_tools.cpp

```diff
@@ -273,7 +277,8 @@ void costSourceToMsg(const CostSource& cost_source, moveit_msgs::msg::CostSource
 void contactToMsg(const Contact& contact, moveit_msgs::msg::ContactInformation& msg)
 {
   msg.position = tf2::toMsg(contact.pos);
-  tf2::toMsg(contact.normal, msg.normal);
+  // tf2::toMsg(contact.normal, msg.normal);
+  tf2::toMsg(contact.normal);
```

Comment:
@anasarrak, are you sure this call replace the previous one? shouldn't it be:
```suggestion
msg.normal =  tf2::toMsg(contact.normal);
```

Reply:
I think that this change was made when we had the problems with Eigen..

Reply:
I suggest this changes since msg.normal is a geometry_msgs/Vector3

```
  geometry_msgs::msg::Vector3 geo_msg;
  geo_msg = msg.normal;

  msg.normal = geo_msg;
```
The suggestion that you've made is failing since it expects a geometry_msg.